example/l2fwd_simple/odp_l2fwd_simple.c

```diff
@@ -160,6 +164,8 @@ int main(int argc, char **argv)
 		       " and source MAC address\n");
 		exit(1);
 	}
+	if (argc == 6 && !strncmp(argv[5], "-t", 2))
```

Comment:
As per previous comments, maybe allow -t to be an option that takes an optional parameter (defaulting to 1). This would mean how many seconds without traffic from pktin to wait before exiting the loop.

Reply:
yep, that also good thing to do.